src/main/java/seedu/address/logic/parser/CliSyntax.java

```diff
@@ -11,5 +11,5 @@
     public static final Prefix PREFIX_EMAIL = new Prefix("e/");
     public static final Prefix PREFIX_ADDRESS = new Prefix("a/");
     public static final Prefix PREFIX_TAG = new Prefix("t/");
-
+    public static final Prefix PREFIX_PATH = new Prefix("pa/");
```

Comment:
This prefix seems unnecessary. It is not in your MESSAGE_USAGE string nor is it used in parser. No prefix is fine, just like delete and find commands

Reply:
Yeap will clean it by the end! Still working on it